install/_install.cmake

```diff
@@ -64,7 +64,7 @@ elseif(APPLE)
             DESTINATION .)
 
     install(TARGETS ${PROJECT_NAME}_dmx_preview ${PROJECT_NAME}_mdl_preview
-            DESTINATION "${PROJECT_NAME_PRETTY}.app/Contents/PlugIns/previews")
+            DESTINATION "${PROJECT_NAME}.app/Contents/PlugIns/previews")
```

Comment:
Ditto, the bundle is named after the capitalized variant of the name? I checked the installer output and this line worked